restoration-ecology.csl

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" default-locale="en-GB" demote-non-dropping-particle="sort-only" version="1.0" class="in-text">
+  <info>
+    <title>Restoration Ecology</title>
+    <id>http://www.zotero.org/styles/restoration-ecology.csl</id>
+    <link href="http://www.zotero.org/styles/modern-humanities-research-association" rel="self"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1111/(ISSN)1526-100X/homepage/ForAuthors.html" rel="documentation"/>
+    <author>
+      <name>Caitlin Rottler</name>
+      <email>caitlin.m.rottler@gmail.com</email>
+    </author>
+    <link href="https://www.zotero.org/styles/genome-biology-and-evolution" rel="template"/>
```

Comment:
This should be `<link href="http://www.zotero.org/styles/genome-biology-and-evolution" rel="template"/>` ("http" instead of "https")